packages/gatsby-plugin-emotion/src/gatsby-node.js

```diff
@@ -1,8 +1,11 @@
-exports.modifyBabelrc = ({ babelrc }) => {
+exports.modifyBabelrc = ({ babelrc }, pluginOptions) => {
   if (process.env.NODE_ENV !== `production`) {
     return {
       plugins: [
-        [require.resolve(`babel-plugin-emotion`), { sourceMap: true }],
+        [
+          require.resolve(`babel-plugin-emotion`),
+          pluginOptions ? pluginOptions : { sourceMap: true, autoLabel: true },
```

Comment:
Instead of using straight `pluginOptions` I would merge default options with user defined options. It would seem weird if user adds unrelated option and suddenly `autoLabel` is turned off IMO

Reply:
Sounds great. Is the object spread operator available at this point or should I use Object.assign()?

Reply:
spread operator will work here - we do run babel on `src` files to produce builds